warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -826,12 +826,16 @@ protected Configuration parseArguments(String[] args, Configuration conf) throws
      */
     protected void configureBulkPartitionerAndOutputFormatter(Job job, AccumuloHelper cbHelper, Configuration conf, FileSystem outputFs)
                     throws AccumuloSecurityException, AccumuloException, IOException, URISyntaxException, TableExistsException, TableNotFoundException {
-        if (null == conf.get("split.work.dir")) {
-            conf.set("split.work.dir", conf.get("ingest.work.dir.qualified"));
+        if (null == conf.get(SplitsFile.SPLIT_WORK_DIR)) {
+            conf.set(SplitsFile.SPLIT_WORK_DIR, conf.get("ingest.work.dir.qualified"));
         }
         conf.setInt("splits.num.reduce", this.reduceTasks);
         // used by the output formatter and the sharded partitioner
-        ShardedTableMapFile.setupFile(conf);
+        long before = System.currentTimeMillis();
+        SplitsFile.setupFile(job, conf);
+        long after = System.currentTimeMillis();
```

Comment:
This could be a follow on task, but perhaps we should make use of org.springframework.util.StopWatch class to make this prettier. StopWatch has a nice prettyPrint method that will show the percentage time for each stage too.

Reply:
agreed

Reply:
not changing that right now...